CHANGELOG.md

```diff
@@ -1,6 +1,247 @@
 Changelog
 =========
 
+1.8.0
+-----
+
+### IMPORTANT NOTES
+
+#### Last release before a PHP minimum version bump
+
+  Release 1.8.0 will be the last release with compatibility for PHP 5.2 - 5.5. With the next release (v2.0.0), the minimum PHP version will be bumped to 5.6+, matching the current PHP minimum version requirement of WordPress Core.
```

Comment:
Yes, you are right. I thought about not mentioning it, but then again, this is now an official part of the WordPress project since it was moved.

But I'll remove this mention, as it doesn't really add any meaningful info anyway.